ee/clickhouse/sql/session_recording_metadata.py

```diff
@@ -0,0 +1,94 @@
+from django.conf import settings
+
+from ee.clickhouse.sql.clickhouse import KAFKA_COLUMNS, kafka_engine
+from ee.clickhouse.sql.table_engines import Distributed, ReplacingMergeTree, ReplicationScheme
+from ee.kafka_client.topics import KAFKA_SESSION_RECORDING_METADATA
+
+SESSION_RECORDING_METADATA_DATA_TABLE = (
+    lambda: "sharded_session_recording_metadata" if settings.CLICKHOUSE_REPLICATION else "session_recording_metadata"
+)
+
+SESSION_RECORDING_METADATA_TABLE_BASE_SQL = """
+CREATE TABLE IF NOT EXISTS {table_name} ON CLUSTER '{cluster}'
+(
+    uuid UUID,
+    timestamp DateTime64(6, 'UTC'),
```

Comment:
What does this timestamp represent?

Reply:
As with UUID. It's the snapshot event timestamp. Not applicable here!